addusers.py

```diff
@@ -334,9 +382,13 @@ def parse_rows(rows, select_user=None):
     parser = argparse.ArgumentParser(description=help_description)
     parser.add_argument('-c', '--config',
                         help='Specify configuration file.')
+    parser.add_argument('--m', dest='manual_input', action='store_true',
+                        help='Process manual inputs.')
     parser.add_argument('--debug', action='store_true',
                         help='Print additional debugging output.')
     mode = parser.add_mutually_exclusive_group(required=True)
+    mode.add_argument('--t', dest='test', action='store_true',
+                      help='Process requests for a single user.')
```

Comment:
`help = Testing changes to the code that don't involve the form`

I misinterpreted what kind of tests you were looking for.